### ISSUE TITLE
If you get the best result, I need your help☺️

### ISSUE THREAD

**Issue by Ien001** [songdejia/DFL#8]
Thanks for clicking into this issue!

If you get 85.14% as the final accuracy, as reported in the homepage, would you please share **the loss value** when the program reached the best result, **the initial learning rate**, and **the transform of the data**?😊😊

I only get **78.6%** at best. Every time I run my program, it seems like I will get random results, ranging from 74.0% ~ 78.6%.

**The updates I have done**:
Resnet-50 based model & no-random initialization

**Comment by guyibang** [comment]
l look forward to the non-random initialization part too

**Comment by Ien001** [comment]
@wangkangnian 
I used CUB-200-2011 dataset.
I will post my part of no-random initialization, if it helps.

**Comment by Ien001** [comment]
@wangkangnian 
I have posted it.
I wish it may help you a bit.😊

**Comment by Ien001** [comment]
@wangkangnian @guyibang 
I have posted my part of non-random initialization.
I wish it may help you guys a bit.😊

**Comment by XIELeo** [comment]
would you share the non-random initialization part？thank you！ @wangkangnian

**Comment by wjtan99** [comment]
@Ien001 thanks for your share, I tried it but still could not get top1 acc > 0.85.  What tricks did you use in terms of training parameters and lr strategy?

**Comment by limengyang9368** [comment]
@Ien001 could you please share you initialization part? it really confused me a couple days...

**Comment by APARNA-AMBARAPU** [comment]
> The accuracy seems be limited under 79% whatever i do anything..... And i find mine non-random initialization implementation doesn't work fine, which accuracy is lower than without it.
> Could you post your implementation of non-random initialization?
> Besides, what your dataset is? When i use CUB2010+2011, accuracy increases a lot.

Hi @wangkangnian Can you share your weights where you got 79% accuracy? I could only reach 52% at best. If you could not share the weights, could you let me know the changes in code you made w.r.t the repo? Thanks!!
